src/Serilog.HttpClient/LoggingDelegatingHandler.cs

```diff
@@ -203,22 +187,26 @@ static double GetElapsedMilliseconds(long start, long stop)
                     }
                 }
 
-                var responseData = new
-                {
-                    StatusCode = (int?)resp?.StatusCode,
-                    IsSucceed = isRequestOk,
-                    ElapsedMilliseconds = elapsedMs,
-                    BodyString = responseBodyText,
-                    Body = responseBody,
-                    Header = responseHeader,
-                };
-
-                _logger.Write(level, ex, _options.MessageTemplate, new
+                using (Serilog.Context.LogContext.PushProperty("RequestMethod", req.Method))
+                using (Serilog.Context.LogContext.PushProperty("RequestUri", req.RequestUri))
+                using (Serilog.Context.LogContext.PushProperty("RequestScheme", req.RequestUri.Scheme))
+                using (Serilog.Context.LogContext.PushProperty("RequestHost", req.RequestUri.Host))
+                using (Serilog.Context.LogContext.PushProperty("RequestPath", req.RequestUri.AbsolutePath))
+                using (Serilog.Context.LogContext.PushProperty("RequestQueryString", req.RequestUri.Query))
+                using (Serilog.Context.LogContext.PushProperty("RequestQuery", requestQuery))
```

Comment:
I'm not sure why we had both `QueryString` and `Query`. Perhaps there's some value in deserializing the query string but I wonder if having both is redundant. FWIW, I would just stick with QueryString.

Reply:
Deserializing query string as Query property usefull when you want to search based on query string properties as structural data in platforms like splunk, elastic or similar ones.